core/models.py

```diff
@@ -192,6 +213,78 @@ def delete(self):
         self.is_deleted = True
         self.save()
 
+    def add_default_content(self):
```

Comment:
I think this should be separated from the model. We should run it once per every new instance, keeping it in the model might end up calling it by mistake. Keeping it as a function separated from the event would be enough, I think :)